recipes/zeromq/all/conandata.yml

```
@@ -9,14 +12,37 @@ sources:
     url: "https://github.com/zeromq/libzmq/releases/download/v4.3.2/zeromq-4.3.2.tar.gz"
     sha256: "ebd7b5c830d6428956b67a0454a7f8cbed1de74b3b01e5c33c5378e22740f763"
 patches:
+  "4.3.5":
+    - patch_file: "patches/0003-rpath-macos-4.3.5.patch"
+      patch_description: "hardcoded full install path on local machine"
```

```suggestion
      patch_description: "hardcoded full install path on local machine"
```

well it's the opposite, this patch removes hardcoded full install path